vignettes/visualization-vignette.Rmd

```diff
@@ -0,0 +1,93 @@
+---
+title: "visualization-vignette"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{visualization-vignette}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+```{r setup}
+library(mlr3fairness)
+library(mlr3learners)
+library(mlr3)
+library(ggplot2)
+library(data.table)
+```
+
+# Why we need fairness visualizations:
+
+Fairness Visualizations are the communications. Most of the users are not expected to understand fairness problems from fairness metrics. They are just numbers for non-experts. However, through fairness visualizations we could provide a clear, thorough and understandable way to understand the fairness problems. More than that, users could quickly diagnose their models through visualizations and locate the fairness problems. In this vignette we will showcase some of the pre build fairness visualization functions.
```

Comment:
Really run a spell check, grammar check and capitalization check.

Reply:
Done for the whole visualization vignettes